### PR TITLE
Add lists to HTML emails

### PR DESCRIPTION
**Pull request by nickvergessen**
Required for https://github.com/nextcloud/activity/pull/141

### CODE REVIEW

File: lib/public/Mail/IEMailTemplate.php [hunk 1]
```diff
@@ -81,6 +81,16 @@ public function addHeading($title, $plainTitle = '');
...
 
 	/**
+	 * Adds a list item to the body of the email
+	 *
+	 * @param string $text
+	 * @param string $plainText Text that is used in the plain text email
+	 *   if empty the $text is used
+	 * @since 12.0.0
+	 */
+	public function addBodyListItem($text, $plainText = '');
```
Comment (MorrisJobke) [inline lib/public/Mail/IEMailTemplate.php]: Shouldn't this be a bit more sophisticated? Like "Main text" and then meta text (for the date for example) and an optional icon? (for small file previews)

Comment (nickvergessen) [inline lib/public/Mail/IEMailTemplate.php]: Well it's a simple list, not one trimmed down so it only works for activity emails :/

Comment (MorrisJobke) [inline lib/public/Mail/IEMailTemplate.php]: Why? I think list can have this optional values or not (meta is shown in small above the main text if present) and the icon is also optionally shown if present. This also could be a list of users with avatars or something like that. If both are not supplied it's a plain list - with it, it's a bit more sophisticated and nice looking.

**Review by MorrisJobke — APPROVED** [review]
Looks good and makes sense - also works with the activity email 👍

**Review by rullzer — APPROVED** [review]
LGTM

**Pull request merged**